docs/ossm/kiali/README.md

```diff
@@ -0,0 +1,463 @@
+# Kiali
+
+Kiali can be used to view the data flow through your application once it has been added to the service mesh.
+
+## About Kiali
+
+Kiali provides observability into the service mesh running on OpenShift. Kiali helps you define, validate, and observe your Istio service mesh. It helps you to understand the structure of your service mesh by inferring the topology, and also provides information about the health of your service mesh.
+
+Kiali provides an interactive graph view of your namespace in real time that provides visibility into features like circuit breakers, request rates, latency, and even graphs of traffic flows. Kiali offers insights about components at different levels, from Applications to Services and Workloads, and can display the interactions with contextual information and charts on the selected graph node or edge. Kiali also provides the ability to validate your Istio configurations, such as gateways, destination rules, virtual services, mesh policies, and more. Kiali provides detailed metrics, and a basic Grafana integration is available for advanced queries. Distributed tracing is provided by integrating Tempo into the Kiali console.
+
+### Kiali Architecture
+
+Kiali is based on the open source [Kiali project](https://www.kiali.io/). Kiali is composed of two components: the Kiali application and the Kiali console.
+
+* **Kiali application (back end)** – This component runs in the container application platform and communicates with the service mesh components, retrieves and processes data, and exposes this data to the console. The Kiali application does not need storage. When deploying the application to a cluster, configurations are set in ConfigMaps and secrets.
+
+* **Kiali console (front end)** – The Kiali console is a web application. The Kiali application serves the Kiali console, which then queries the back end for data to present it to the user.
+
+In addition, Kiali depends on external services and components provided by the container application platform and Istio.
+
+* **Red Hat Service Mesh (Istio)** - Istio is a Kiali requirement. Istio is the component that provides and controls the service mesh. Although Kiali and Istio can be installed separately, Kiali depends on Istio and will not work if it is not present. Kiali needs to retrieve Istio data and configurations, which are exposed through Prometheus and the cluster API.
+
+* **Prometheus** - When Istio telemetry is enabled, metrics data are stored in Prometheus. Kiali uses this Prometheus data to determine the mesh topology, display metrics, calculate health, show possible problems, and so on. Kiali communicates directly with Prometheus and assumes the data schema used by Istio Telemetry. Prometheus is an Istio dependency and a hard dependency for Kiali, and many of Kiali's features will not work without Prometheus.
```

Comment:
Here I think we should make it clearer that Prometheus is an optional dependency for Istio (users do not have to install or integrate  Prometheus for Istio to work) but if they do not do that,  most of Kiali will not work.

```suggestion
* **Prometheus** - When Istio telemetry is enabled, metrics data are stored in Prometheus. Kiali uses this Prometheus data to determine the mesh topology, display metrics, calculate health, show possible problems, and so on. Kiali communicates directly with Prometheus and assumes the data schema used by Istio Telemetry. Prometheus is an Istio optional dependency, but a hard dependency for Kiali, and many of Kiali's features will not work without Prometheus.
```